app/views/kuroko2/job_definition_revisions/index.html.slim

```diff
@@ -0,0 +1,24 @@
+- content_for :title, "##{@definition.id} #{@definition.name} « Job Definitions"
+- content_for :content_title, "Job Definition Revisions"
+
+- content_for :breadcrumb do
+  ol.breadcrumb
+    li= link_to raw('<i class="fa fa-database"></i> Job Definitions'), job_definitions_path
+    li= link_to "Details", @definition
+    li.active Revisions
+
+style= Diffy::CSS
+
+- @revisions.each.with_index do |revision, i|
```

Comment:
(maybe) Use `each_cons(2)` instead of this.
then `html_diff` doesn't need to allow `nil` case, is it right?

Reply:
If using `each_cons(2)`, first revision will not displayed.

Reply:
Oh, my comment was not proper. I should say why you want to show the oldest revision. (weak opinion)
It's better to use `each_with_index(1)` if you let it leave.

Reply:
Showing the oldest revision explains initial script and who created the definition. I think this is helpful.